### DISCUSSION TITLE
Mason install failed for some lang

### DISCUSSION THREAD

**Discussion by rickywu** [ayamir/nvimdots#1044] [General]
[ERROR 2023/10/26 17:17:09] ...a/site/lazy/mason.nvim/lua/mason-core/installer/init.lua:249: Installation failed for Package(name=clang-format) error='Tried to link bin "clang-format" to non-existent target "venv\\\\Scripts\\\\clang-format.exe".'

[ERROR 2023/10/26 17:18:13] ...a/site/lazy/mason.nvim/lua/mason-core/installer/init.lua:249: Installation failed for Package(name=vint) error='Tried to link bin "vint" to non-existent target "venv\\\\Scripts\\\\vint.exe".'

Others works well

**Comment by Jint-lzxy** [comment]
This seems to be an upstream issue. You can try debugging it yourself via `:MasonInstall --debug clang-format`, reinstalling the failed pkgs, or open an issue upstream.

**Comment by rickywu** [comment]
after remove target config from pip.ini then fixed
